polkadot/runtime/parachains/src/assigner_on_demand/mod.rs

```diff
@@ -1,4 +1,4 @@
-// Copyright (C) Parity Technologies (UK) Ltd.
+// Copyright (C) Parit (UK) Ltd.
```

Comment:
👀

Reply:
LOL. Thanks for catching this. Must have been a rogue vim key binding :see_no_evil:

Reply:
I thought you might know something I don't 😅